manifests/cf-manifest/spec/scripts/rotate_vars_store_secrets_spec.rb

```diff
@@ -269,7 +269,7 @@
     it "onlies delete _old secrets for secrets set to rotate" do
       vars_to_rotate = vars_store.keys.select { |k| k.end_with? "_one" }
 
-      rotated_vars_store = rotate(manifest, vars_store, vars_to_rotate: vars_to_rotate, delete: true)
```

Comment:
:confused: This is the only transformation I don't immediately understand.

Reply:
Wait.. does omitting the value just bind it to the same-named local variable in ruby 3?

Reply:
I have no idea, honestly. I allowed Rubocop to fix thing itself and it looks like something it sorted for itself; `bundle exec rubocop -A`